web-testbed/src/testbed/app.py

```diff
@@ -0,0 +1,22 @@
+import toga
+from toga.style import Pack
+from toga.style.pack import COLUMN
+
+from .web_test_harness import WebTestHarness
+
+
+class HelloWorld(toga.App):
+    def startup(self):
+        main_box = toga.Box(style=Pack(direction=COLUMN))
+        self.label = toga.Label(id="myLabel", text="Test App - Toga Web Testing")
+
+        self.web_test = WebTestHarness(self)

```

Comment:
This is a really good piece of refactoring. Longer term, it suggests that there might be an opportunity to move the `WebTestHarness` to a platform-specific "remote control harness" - something that the *backend* (or maybe the probe) would provide to allow an app to be remote controlled.